tests/test_external_search.py

```diff
@@ -863,6 +886,8 @@ class TestExactMatches(ExternalSearchTest):
 
     def setup(self):
         super(TestExactMatches, self).setup()
+        if not self.search:
+            return
```

Comment:
These classes should have been skipped when Elasticsearch wasn't configured, but they were being run.